src/initialize.js

```diff
@@ -157,6 +159,41 @@ export async function runtimeConfig() {
   }
 }
 
+/*
+ * Set custom colors based on the config content.
+ * This method allows to change primary colors and its levels on runtime,
+ * if a specific level is already in the configuration that level will have
+ * priority otherwise the level will be calculated based on primary color by
+ * using the mix function.
+ */
+export async function setCustomPrimaryColors() {
```

Comment:
This function doesn't have any process `async` then you should remove this world